spacy/ml/models/span_finder.py

```diff
@@ -0,0 +1,43 @@
+from typing import Callable, List, Tuple
+
+from thinc.api import Model, chain, with_array
+from thinc.types import Floats1d, Floats2d
+
+from spacy.tokens import Doc
+
+from ...util import registry
+
+InT = List[Doc]
+OutT = Floats2d
+
+
+@registry.architectures("spacy.SpanFinder.v1")
+def build_finder_model(
+    tok2vec: Model[InT, List[Floats2d]], scorer: Model[OutT, OutT]
+) -> Model[InT, OutT]:
+
+    logistic_layer: Model[List[Floats2d], List[Floats2d]] = with_array(scorer)
+    model: Model[InT, OutT] = chain(tok2vec, logistic_layer, flattener())
+    model.set_ref("tok2vec", tok2vec)
+    # XXX Why do we have reference for both the scorer and it being wrapped in with_array?
```

Comment:
What is the status of this comment?

Reply:
Nothing really. Its unclear to me when we decide to add references or not, but it doesn't really matter actually so I'll just remove it!